CONTRIBUTING.md

```diff
@@ -238,6 +238,120 @@ installer-test-rhel-v7-install-default> Formatting './disk.qcow2', fmt=qcow2 clu
 
 </details>
 
+## Container tests
```

Comment:
I think it would be valuable to explicitly mention that these tests are only supported on `x86_64-linux`. It's implied through the attr path, but folks newer to Nix might not understand that and be confused.